packages/create-block/lib/prompts.js

```diff
@@ -79,6 +79,19 @@ const category = {
 	choices: [ 'text', 'media', 'design', 'widgets', 'theme', 'embed' ],
 };
 
+const textdomain = {
+	type: 'input',
+	name: 'textdomain',
+	message: 'The text domain used to internationalize text in the block:',
```

Comment:
Again, the question is whether this setting should apply primarily to the plugin or block.